apps/system/js/sync/places_adapter.js

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* globals appWindowManager */
+/* exported PlacesAdapter */
+
+'use strict';
+
+(function(exports) {
+  var PlacesAdapter = {
```

Comment:
This is the code that is shared between the System app and the History adapter, right? If you're newly adding this code in the systems app, should it not also be removed somewhere? Aren't `addPlace` and `addPlaces` already there?